examples/models/role_playing_with_cohere.py

```diff
@@ -0,0 +1,147 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import os
+
+import cohere
+from cohere.core.api_error import ApiError
+from colorama import Fore
+
+from camel.configs import CohereConfig
+from camel.models import ModelFactory
+from camel.societies import RolePlaying
+from camel.types import ModelPlatformType, ModelType
+from camel.utils import print_text_animated
+
+
+def test_cohere_api(api_key):
```

Comment:
Just curious, I saw that assert was not used in the test. Is it because the results are unpredictable?

Reply:
thanks @koch3092 , this is not test but an example, but we indeed missing test file, @NeilJohnson0930 you can refer to test code in test/models/test_mistral_model.py

Reply:
Got it! test_cohere_model.py will be committed as soon as possible